typescript/formsApps.d.ts

```diff
@@ -356,6 +356,8 @@ export type FormsAppUserBase = {
 }
 
 export type NewFormsAppUser = FormsAppUserBase & {
+  firstName?: string
```

Comment:
Should they be on `FormsAppUserBase` instead?

Reply:
hmm good point